src/welcome.js

```diff
@@ -20,12 +20,12 @@ const Welcome = () => {
             const { tag, label } = x;
```

Comment:
- [ ] I kindly suggest that you organize your folders better. Your `categorySlice.js` and `categoryIndex.js` are not to be inside your redux folder, along with your test folders and files. they supposed to be in your components folder. Pls see below for reference on how to organize your files and folders:

```
app-root-directory
  |- package.json
  |- README.md
  |- /public
    |- index.html
  |- /src
    |- app.css
    |- index.js
    |- /components
    |- /container
    |- /pages
    |- /redux
  |- /node_modules
```

Reply:
Noted and resolved!